config.yaml

```diff
@@ -9,7 +9,7 @@ options:
   git-sync::branch:
     type: string
     description: branch to sync
-    default: ""
+    default: main
```

Comment:
Is that a gitops SOP?
If not, I am inclined to remove the default.

Reply:
Pretty much yes nowadays.